modules/oc-mirror-oci-format.adoc

```diff
@@ -0,0 +1,130 @@
+<4> Specify the full path to a directory on disk where you want to output the catalog. The path must start with `oci://`. This procedure assumes that the directory is named `my-oci-catalog`.
++
+[NOTE]
+====
+You can optionally use the `--oci-registries-config` flag to specify the path to a TOML-formatted `registries.conf` file. You can use this to mirror from a different registry, such as a pre-production location for testing, without having to change the image set configuration file.
+
+.Example registries.conf file
+[source,toml]
+----
+[[registry]]
+ location = "registry.redhat.io:5000"
+ insecure = false
+ blocked = false
+ mirror-by-digest-only = true
+ prefix = ""
+ [[registry.mirror]]
+    location = "preprod-registry.example.com"
+    insecure = false
+----
+
+Set the `location` field in the `registry.mirror` section to an alternative registry location that you want to pull images from. The `location` field in the `registry` section must be the same registry location as the one you specify in the image set configuration file.
+====
+
+. List your directory contents and verify that the following directories were created:
++
+[source,terminal]
+----
+$ ls -l
+----
++
+.Example output
+[source,terminal]
+----
+my-oci-catalog      <1>
+oc-mirror-workspace <2>
+olm_artifacts       <3>
+----
+<1> Directory that contains the OCI catalog. This procedure assumes that it is named `my-oci-catalog`.
+<2> Directory that contains each image in the catalog in its original format.
+<3> Directory that contains the files that describe the Operator bundles that this catalog references.
+
+. Update the image set configuration file to specify the location of the catalog on disk to mirror to the target mirror registry:
++
+.Example image set configuration file for mirroring to mirror registry
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+mirror:
+ operators:
+ - catalog: oci:///home/user/oc-mirror/my-oci-catalog/redhat-operator-index <1>
+   packages:
+   - name: aws-load-balancer-operator
+----
+<1> Specify the absolute path to the location of the OCI catalog on disk. This procedure assumes that you used `my-oci-catalog` as the directory and mirrored the `redhat-operator-index` catalog. The path must start with `oci://`.
++
+// TODO: mention about multiple catalogs...
+
+. Run the oc mirror command to process the image set file on disk and mirror the contents to a target mirror registry:
++
+[source,terminal]
+----
+$ oc mirror --config=./imageset-config.yaml \ <1>
+  --use-oci-feature \                         <2>
+  --oci-feature-action=mirror \               <3>
+  docker://registry.example:5000              <4>
+----
+<1> Pass in the updated image set configuration file. This procedure assumes that it is named `imageset-config.yaml`.
+<2> Use the `--use-oci-feature` flag to enable the OCI feature.
+<3> Set the `--oci-feature-action` flag to `mirror` to mirror the catalog to the target mirror registry.
```

Comment:
I actually do need both since one is a value and one is a verb, but I'm glad you pointed it out because it does look odd back to back. I will rework the sentences!